### DISCUSSION TITLE
[Support]: Right way to label images in Plus and more

### DISCUSSION THREAD

**Discussion by jiiins** [blakeblackshear/frigate#15252] [General Support]
### Describe the problem you are having

Q1: I'm labeling images in Plus and I'm not sure about the right way for situations like this:

![image](https://github.com/user-attachments/assets/c7b0028b-347b-4ef1-b1af-30dfbec34aa7)

In this case, I indicated that it's not a person, with the idea of drawing the correct box. Is this the right approach? Or is it better to say it's a person and then adjust the box accordingly?

Q2: is it possible to go back to the previous image? It happens sometimes that I notice I missed out an object just when I hit space...

Q3: in many cases, the image contains more objects (rightly detected in Frigate), but the image for Plus only shows 1, and I have to draw all the missing boxes all the time. Is there a way to avoid this?

Thanks a lot

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
.
```

### Relevant Frigate log output

```shell
.
```

### Relevant go2rtc log output

```shell
.
```

### FFprobe output from your camera

```shell
.
```

### Install method

Proxmox via Docker

### docker-compose file or Docker CLI command

```yaml
.
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

.

**Comment by NickM-27** [comment]
1. Say it is a person and adjust the box accordingly
2. You can use the browser back button
3. There is a suggestions feature being worked on that will label other objects
